pypeit/slittrace.py

```diff
@@ -68,7 +68,7 @@ def exclude_for_reducing(self):
     def exclude_for_flexure(self):
         # Ignore these flags when performing a flexure calculation
         #  Currently they are *all* of the flags..
-        return ['SHORTSLIT', 'USERIGNORE', 'BADWVCALIB', 'BADTILTCALIB', 'BADALIGNCALIB',
+        return ['SHORTSLIT', 'BOXSLIT', 'USERIGNORE', 'BADWVCALIB', 'BADTILTCALIB', 'BADALIGNCALIB',
```

Comment:
I'm nervous that the order of these matters
for backwards compatibility.  Admittedly, we
aren't very backwards compatible, but am 
raising it as a concern.

Is there a good reason why we have put `BOXSLIT` second?

Reply:
I'm not sure if @debora-pe had a specific reason for putting it second in the list, but it shouldn't matter to the determination of the flagged bits, and I don't think it should affect backwards compatibility.

Reply:
No specific reason besides trying to keep visually the same order that is in `SlitTraceBitMask.__init__()`. I also thought it would not affect backwards compatibility.